test/al_aws_collector_test.js

```diff
@@ -142,9 +143,24 @@ describe('al_aws_collector tests', function(done) {
     it('process success', function(done) {
         AlAwsCollector.load().then(function(creds) {
             var collector = new AlAwsCollector(
-            context, 'cwe', AlAwsCollector.IngestTypes.SECMSGS, '1.0.0', creds, formatFun);
-                collector.process('some-data', {}, function(error) {
-                done();
+                context, 'cwe', AlAwsCollector.IngestTypes.SECMSGS, '1.0.0', creds, formatFun);
+            var data = 'some-data';
+            collector.process('some-data', {}, function(error) {
+                assert.ifError(error);
+                sinon.assert.called(alserviceStub.post);
+                zlib.deflate(data, function(compressionErr, compressed) {
+                    assert.ifError(compressionErr);
+                    var url = '/data/secmsgs';
+                    var body = { json: false,
```

Comment:
Let's just mock al-collector-js `IngestC` class and make sure `sendSecmsgs` is called with correct parameters, rather than checking the internals. We need to avoid broken test here because of some changes being made in al-collector-js.